02/code-monk08/game.py

```diff
@@ -0,0 +1,84 @@
+import itertools
+import random
+from data import DICTIONARY, LETTER_SCORES, POUCH
+
+
+NUM_LETTERS = 7
+
+
+def draw_letters():
+    """Pick NUM_LETTERS letters randomly. Hint: use stdlib random"""
+    return random.choices(POUCH, k=NUM_LETTERS)
+
+
+def input_word(draw):
+    """Ask player for a word and validate against draw.
+    Use _validation(word, draw) helper."""
+    while True:
+        enteredword = input(f'enter the valid word from the letters you have drawn {draw}').upper()
```

Comment:
better entered_word

Reply:
okay, I will remember naming variable like this.